sites/authentication/_base.scss

```diff
@@ -7,6 +7,10 @@
 	.btn {
 		border-radius: 0px;
 		margin-top: 5px;
+
+		.home & {
```

Comment:
This belong to the home page template not to the authentication generic pattern.

File [home.css](https://github.com/wet-boew/GCWeb/blob/master/templates/home/_base.scss#L12)

.home #wb-so .btn {
   margin-top: 3px;
}

Reply:
Le SASS actuel est compilé de la manière suivante en CSS: 

`#wb-so .btn {
  border-radius: 0px;
  margin-top: 5px;
}
.home #wb-so .btn {
  margin-top: 3px;
}
#wb-so a.btn:hover {
  background-color: #444;
}`

Il permet donc d'effectuer la même commande, mais plus simplement.